library/pk_wrap.c

```diff
@@ -1162,8 +1162,13 @@ static int ecdsa_sign_wrap( void *ctx_arg, mbedtls_md_type_t md_alg,
     size_t key_len;
     unsigned char buf[MBEDTLS_PK_ECP_PRV_DER_MAX_BYTES];
     unsigned char *p;
+#if defined(MBEDTLS_ECDSA_DETERMINISTIC)
```

Comment:
Commit c0b47d5e87290ea83ff7a5e00c177b64f8300736 has a couple of misspellings:
- `Use deterministic in PSA when we do in legacy` <- deterministic ECDSA?
- `The failure was cause by` <- caused
It would be great to know what test function(s?) you're talking about to look at more details.

Reply:
This is still present, but is a minor.